administrator/components/com_associations/views/associations/tmpl/default.php

```diff
@@ -22,6 +23,61 @@
 $input = $app->input;
 
 $assoc = JLanguageAssociations::isEnabled();
+
+// print_r($input->post);
+
+$component = $input->post->get('component');
+$language = $input->post->get('ref-language');
+
```

Comment:
True that, but as JM said before, all components have different fields, so we must fetch them somehow and have the ID, Title, Associations and Language in common.